aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -526,6 +532,32 @@ Resources:
 <%  unless load_balancer -%>
           - !ImportValue VPC-ELBSecurityGroup # Accept HTTP traffic on daemon
 <%  end -%>
+  DaemonLowDiskspaceAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      AlarmActions:
+        - arn:aws:sns:us-east-1:475661607190:CDO-Urgent
+      AlarmDescription: Send page if daemon disk space utilization exceeds 80% for an hour
+      AlarmName: DaemonLowDiskSpace
+      ComparisonOperator: GreaterThanThreshold
+      Dimensions:
+        - Name: Filesystem
+          Value: '/dev/xvda1'
+        - Name: MountPath
+          Value: '/'
+        - Name: InstanceId
+<% if daemon_instance_id -%>
+          Value: <%=daemon_instance_id%>
+<% else -%>
+          Value: !Ref <%=daemon%>
+<% end -%>
```

Comment:
this could be reduced to a one-liner using the [`<%=` expression-printing tag](https://puppet.com/docs/puppet/5.3/lang_template_erb.html#expression-printing-tags): `Value: <%= daemon_instance_id || "!Ref #{daemon}" %>`